_includes/linkers/meetings-pages.md

```diff
@@ -0,0 +1,18 @@
+{% capture /dev/null %}
+{% if page.url == "/meetings/" %}
+  {% assign _index_links = _index_links | append: "<span style='font-weight: bold'>By date</span>" %}
```

Comment:
These links and stylings aren't working for me. The `page.url` values and the `<span style>` tags both seem to have issues.

Reply:
Proposed fix in #29.

Reply:
Thanks!